README.md

```diff
@@ -175,6 +175,16 @@ private computeBaz(fooChangeRecord: object) {
 }
 ```
 
+Also note that you may combine `@property` and `@computed` for additional
+configuration. This is especially useful to apply a `type` without needing
+the metadata API.
+
+```ts
+@property({type: String})
+@computed<MyElement>('foo', 'bar')
```

Comment:
I think the opposite order is needed, since decorators are applied bottom up. Otherwise you'll get a console warning when `@computed` runs and doesn't find a type.